### ISSUE TITLE
Trailer Preview

### ISSUE THREAD

**Issue by Mkdantas** [Stremio/stremio-web#533]
There isn't an option for trailer preview like most streaming services when you are hovering through their library. 
I tried implementing it, however, this is my first issue ever, please provide me with feedback and guide me on changes i should make if possible.

![Trailer preview)](https://github.com/Stremio/stremio-web/assets/50972667/1a23510e-3240-41c9-a194-12b5448bb30e)

I used the ```ytId``` to embed a youtube video, however, if there is a way to serve the video id like how it is provided in metaDetails i could use stremio's player instead of youtube to show a preview of the trailer.

this is my fork with the above updates committed https://github.com/Mkdantas/stremio-web

**Comment by Iheuzio** [comment]
An alternative could be to use the same trailer url and serve that video.

**Comment by Mkdantas** [comment]
> An alternative could be to use the same trailer url and serve that video.

Yeah, do you know how I can get that url to be in metaItem props? The trailer url present in metaDetail is not present in metaItem

**Comment by Iheuzio** [comment]
> > An alternative could be to use the same trailer url and serve that video.
> 
> Yeah, do you know how I can get that url to be in metaItem props? The trailer url present in metaDetail is not present in metaItem

Nevermind, I see what you're trying to achieve. Its a pretty cool implementation, however for actual general use it would be best to leave the embeded players in their own window, thus the trailer pane is fine as is. 

On mobile this wouldn't work, nor android tvs. It would create a navigation problem for the users.

**Comment by ceejey** [comment]
How does Nuvio do it? Better to implement it in stremio, too.
